changelog/26200.txt

```diff
@@ -0,0 +1,6 @@
+```release-note:bug
+auth/ldap: Fix login error missing entity alias attribute value.
+```
+```release-note:bug
+auth/ldap: Fix login error for group search anonymous bind.
+```
```

Comment:
Does this actually work? If so TIL 😄

Reply:
It does! you can bunch multiple CL entries in a single file.